securesystemslib/storage.py

```diff
@@ -189,9 +189,9 @@ class FilesystemBackend(StorageBackendInterface):
     # objects.
     _instance = None
 
-    def __new__(cls, *args, **kwargs):
+    def __new__(cls) -> FilesystemBackend:
         if cls._instance is None:
-            cls._instance = object.__new__(cls, *args, **kwargs)
+            cls._instance = object.__new__(cls)
         return cls._instance
```

Comment:
if there's a neater way to handle this I'm happy to hear it -- maybe we just want to ignore mypy here since `__new__()` has some magic for silently accepting kwargs...

Reply:
I will do this to keep the PR minimal